README.md

```diff
@@ -17,190 +17,88 @@ By [Fauna, Inc](http://faunadb.com).
-              }
-              // create some indexes to be able to perform queries
-              client.query(
-                Do(exprs:
-                          Create(ref: Ref("indexes"),
-                              params:Obj(["name": "posts_by_tags",
-                                          "source": BlogPost.classRef,
-                                          "terms": Arr(Obj(["field": Arr("data", "tags")])),
-                                          "values": Arr()])),
-                          Create(ref: Ref("indexes"),
-                              params: Obj(["name": "posts_by_name",
-                                           "source": BlogPost.classRef,
-                                           "terms": Arr(Obj(["field": Arr("data", "name")])),
-                                           "values": Arr()]))
-                )
-              , completion: callback)
-          }
-      }
-  }
-}
-
-// Once we've set up fauna scheme we are able to create instances and perform queries among other things...
-
-func createInstances(callback: (Result<Value, Error> -> ())) {
-    client.query({
-        let blogPosts = (1...100).map {
-            BlogPost(name: "Blog Post \($0)", author: "FaunaDB",  content: "content", tags: $0 % 2 == 0 ? ["philosophy", "travel"] : ["travel"])
-        }
-        return Map(collection: Arr(blogPosts)) { blogPost  in
-             Create(ref: Ref("classes/posts"), params: Obj(["data": blogPost]))
-        }
-    }(), completion: callback)
-}
-
-// let's invoke these async functions to set up the scheme and create blog post instances.
-setUpSchema(db_name) { result in
-      guard let _ = try? result.dematerialize() else {
-          // show error message or propagates it
-          return
-      }
-      createInstances { createInstancesR in
-          guard let _ = try? createInstancesR.dematerialize() else { return /* handle error */ }
-          // blog post instances are now saved in FaunaDB
-      }
-}
-
 ```
 
-#### Basic Usage (using RxSwift reactive extensions)
-
-```swift
-
-import Foundation
-import Result
-import FaunaDB
-import RxFaunaDB
+Then we should instantiate a FaunaDB client, which allows us to perform queries.
 
-// create a fauna client
-// Logger will shows in the Xcode console the curl representation of each message sent to FaunaDB, it also shows the fauna response data. This is useful during app development.
+> All communication between your app and FaunaDB db should be done through a Fauna Client, a `Client` instance.
 
+The simplest way to set up a client is by passing Fauna DB secret as a parameter. It also accepts other configurations such as endpoint, timeout and observers.
 
-let secret = <YOUR_CLIENT_SECRET>
-let dbName = <YOUR_DB_NAME>
-var client = Client(secret: secret, observers: [Logger()])
+```swift
+let client = Client(secret: <YOUR_FAUNA_DB_SECRET>)
+```
 
-let disposeBag = DisposeBag()
+> Typically we have only one client instance.
```

Comment:
This sentence seems unnecessary. It's also not true because apps can authenticate specific requests with different keys.